tests/test_integrated_channels/test_xapi/test_serializers.py

```diff
@@ -57,7 +57,10 @@ def test_data(self, mock_third_party_api):
 
         with mock.patch.object(self.user, 'profile', create=True) as mock_user_profile:
             mock_user_profile.country.code = 'PK'
-            assert LearnerInfoSerializer(self.user).data == self.expected_data
+            assert LearnerInfoSerializer(
```

Comment:
We should change the tests to reflect the actual use case. Which is a learner has multiple enterprises and when management command is run to transmit data of a particular enterprise only that enterprise is reflected in user's data.

Reply:
Added another test to verify the expected output in case of user with multiple enterprises